src/main/resource-applier.ts

```diff
@@ -21,56 +21,73 @@
 
 import type { Cluster } from "./cluster";
 import type { KubernetesObject } from "@kubernetes/client-node";
-import { exec } from "child_process";
-import fs from "fs";
+import fse from "fs-extra";
 import * as yaml from "js-yaml";
 import path from "path";
 import * as tempy from "tempy";
 import logger from "./logger";
 import { appEventBus } from "../common/event-bus";
 import { cloneJsonObject } from "../common/utils";
+import { inspect } from "util";
+import { promiseExec } from "./promise-exec";
+
+function sanitize(raw: string): string {
+  return inspect(raw, false, undefined, false);
```

Comment:
Why do we need to `sanitize` with `inspect`?

Reply:
Oh, it was mentioned in description.

Reply:
Yes, though I have now realized that this is incorrect. I will try and find a different way of doing this. 

`inspect` doesn't produce the result we would like if the raw string includes back ticks